Ginger/GingerCoreCommon/Repository/SolutionRepository.cs

```diff
@@ -137,6 +138,11 @@ public void AddRepositoryItem(RepositoryItemBase repositoryItem)
             SRII.ItemRootRepositoryFolder.AddRepositoryItem(repositoryItem);
         }
 
+        public async Task SaveRepositoryItemAsync(RepositoryItemBase repositoryItem)
+        {
+            await Task.Run(() => SaveRepositoryItem(repositoryItem));
```

Comment:
Codacy found an issue: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5970337)